tests/interpolator/CMakeLists.txt

```diff
@@ -6,10 +6,10 @@ include(test_util)
 ################################################################################
 # Tests for the Interpolator class
 
-create_standard_test(NAME fortran_interpolator SOURCES interpolator.c)
```

Comment:
Just a question I probably should've asked earlier: what is a "standard test"?

Reply:
Just a test that links to the `aero` library and has the usual include directories. In cases where we want to use a mock version of some class, we might want to compile differently and then just use the `add_aero_test()` function.